### PR TITLE
Bump PHPUnit version

### PR DESCRIPTION
**Pull request by Sam-Burns**
Increases PHPUnit version to 6.1, now PHP5 no longer required

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
         "php":                      "^7.0",
         "phpspec/prophecy":         "^1.5",
         "phpspec/php-diff":         "^1.0.0",
-        "sebastian/exporter":       "^1.0 || ^2.0 || ^3.0",
+        "sebastian/exporter":       "^3.1",
```
Comment (stof) [inline composer.json]: I would revert this change, as it makes it harder for projects to avoid a compatibility hell (this is a normal requirement, so it affects users). Version 2 of the exporter is also compatible with PHP 7.
As the API of the exporter is the same, there is absolutely no maintenance cost here.

**Comment by ciaranmcnulty** [comment]
If we're not using any of the features I don't see the point TBH
